pkg/plumber/constants.go

```diff
@@ -8,10 +8,10 @@ const (
 	// that the pod is running.
 	PlumberJobTypeLabel = "lighthouse.jenkins-x.io/type"
 	// PlumberJobIDLabel is added in resources created by lighthouse and
-	// carries the ID of the PlumberJob that the pod is fulfilling.
-	// We also name resources after the PlumberJob that spawned them but
+	// carries the ID of the PlumberArguments that the pod is fulfilling.
+	// We also name resources after the PlumberArguments that spawned them but
 	// this allows for multiple resources to be linked to one
-	// PlumberJob.
+	// PlumberArguments.
```

Comment:
Why not `PipelineOptions` instead of `PlumberArguments`? It sounds to me more appropriate for this.

Reply:
sounds good to me - PR on way shortly...